Firmware/ultralcd.cpp

```diff
@@ -5055,11 +5049,11 @@ static void lcd_rename_sheet_menu()
         lcd_encoder = menuData->name[0];
         menuData->initialized = true;
     }
-    if (lcd_encoder < '\x20') lcd_encoder = '\x20';
-    if (lcd_encoder > '\x7F') lcd_encoder = '\x7F';
+    if (lcd_encoder < 0x20) lcd_encoder = 0x20;
+    if (lcd_encoder > 0xFF) lcd_encoder = 0xFF;
```

Comment:
Just adding a note that this should be tested specifically. Can a sheet name now be a complete white block like we use in the MMU filament load test progress bar? :D

Reply:
Yup :P. The entire charset can be used now

Reply:
I was right. I think this is a feature not a bug 😄